river/base/clusterer.py

```diff
@@ -11,14 +11,18 @@ def _supervised(self):
         return False
 
     @abc.abstractmethod
-    def learn_one(self, x: dict) -> "Clusterer":
+    def learn_one(self, x: dict, sample_weight: int = None) -> "Clusterer":
         """Update the model with a set of features `x`.
 
         Parameters
         ----------
         x
             A dictionary of features.
 
+        sample_weight
+            Integer, optional (default=None)
```

Comment:
Don't specify the type and the default value. This should all be done in the signature.

Reply:
Resolved.